.github/workflows/test.yml

```diff
@@ -72,3 +77,16 @@ jobs:
           disable_search: true
           fail_ci_if_error: false
           use_oidc: true
+
+      - name: Install uv
+        uses: astral-sh/setup-uv@e92bafb6253dcd438e0484186d7669ea7a8ca1cc # v6.4.3
+
+      - name: Build Packages
+        run: |
+          uv build
+          uv tool run twine check dist/*
+
+      # Uses to OIDC identification between GitHub and PyPI
```

Comment:
This might require something being toggled on the PyPi side of things to permit OIDC authentication, I'm not sure.

Reply:
Done. It's pinned to the workflow file name, and we haven't setup GitHub Environments.

<img width="437" height="168" alt="Image" src="https://github.com/user-attachments/assets/c7ec78bc-7e6d-4dd7-b4d3-6dd768fb448d" />